join.html

```diff
@@ -26,21 +25,34 @@ <h1 class="title page-title">Join Us!</h1>
             <p>
                 Want to learn more? Check out role responsibilities below to see exactly what Teach LA has in store.
             </p>
+            <h3>
+                <b>We are currently operating entirely virtually!</b>
+            </h3>
```

Comment:
Two comments here:

1. in general, you want HTML to describe the structure of the page, and CSS to describe the styling. In particular, `h3` should be a header of a section, instead of being used for size/font/etc. - instead, use CSS for this! same for `b`!
2. I think we can try to style this a little differently, maybe like an "Alert"-style component? Here's [an example of what that might look like](https://getbootstrap.com/docs/4.0/components/alerts/). That being said, if you're not able to figure it out, don't worry bout it for now!